hummingbot/connector/exchange/paper_trade/paper_trade_exchange.pyx

```diff
@@ -703,7 +773,7 @@ cdef class PaperTradeExchange(ExchangeBase):
                 order_id,
                 base_asset,
                 quote_asset,
-                base_asset if config.sell_fees_asset is AssetType.BASE_CURRENCY else quote_asset,
+                base_asset,
```

Comment:
Is this the correct fee asset? Also please check because the fee amount is always 0, but the real fee paid has been already calculated. We should use that information.